crates/re_log_types/src/data_row.rs

```diff
@@ -546,7 +546,15 @@ impl std::fmt::Display for DataRow {
     fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
         writeln!(f, "Row #{} @ '{}'", self.row_id, self.entity_path)?;
         for (timeline, time) in &self.timepoint {
-            writeln!(f, "- {}: {}", timeline.name(), timeline.typ().format(*time))?;
+            // TODO(paris): Figure how to pass show_timestamps_in_local_timezone in to fmt(). We get:
```

Comment:
as with `IndexedBucket` I don't think we need to care right now about changing the display for `DataRow`. For any more sophisticated ui we have a look at what's inside a row first. Making UTC a good default again.

Reply:
Great thanks, left it in UTC.